flink-runtime/src/test/java/org/apache/flink/runtime/state/ttl/StateBackendTestContext.java

```diff
@@ -34,6 +34,7 @@
 import org.apache.flink.runtime.state.SnapshotResult;
 import org.apache.flink.runtime.state.StateBackend;
 import org.apache.flink.runtime.state.internal.InternalKvState;
+import org.apache.flink.shaded.guava18.com.google.common.collect.Lists;
```

Comment:
If possible, is there an alternative to write the code that let's us avoid this new import?

Reply:
Here we need to create a collection with a single element. I've tried `Collections.singletonList` and `Arrays.asList` but neither is ok since we will call `remove` method of this collection later. So we will need to write more codes by ourselves to get rid of this.

Reply:
I think it is ok to do `new ArrayList(1)` and `add`.